### PR TITLE
github dl badges in README

### PR DESCRIPTION
**Pull request by tooomm**
## What will change with this Pull Request?
- have dl badges again for both channels which show the current version and link to gh releases
- the dev badge also shows pre-releases versions correctly and doesn't link to /latest
- all-time download count
- separate download count for recent release

## Screenshots
before:
old

after:
new

### CODE REVIEW

**Comment by tooomm** [comment]
It would also be possible to have the dl count of just the latest main release behind the corresponding release badge: dl count on latest release

File: README.md [hunk 1]
```diff
@@ -17,20 +17,20 @@
 Cockatrice is an open-source multiplatform supported program for playing tabletop card games over a network. The program's server design prevents any kind of client modifications to gain an unfair advantage in a game. The client also has a built in single-player mode where you can brew without being connected to a server. This project is written in C++ and is using the Qt5 libraries.<br>
 
 
-# Downloads
+# Downloads [![Cockatrice Eternal Download Count](https://img.shields.io/github/downloads/cockatrice/cockatrice/total.svg)](http://somsubhra.github.io/github-release-stats/?username=cockatrice&repository=cockatrice)
```
Comment (ZeldaZach) [inline README.md]: When clicked this just brings you to the shields.io site

Comment (tooomm) [inline README.md]: It works for me with chrome and ff:
https://github.com/Cockatrice/Cockatrice/tree/tooomm-dl_badges#downloads-

Correctly opens http://somsubhra.github.io/github-release-stats/?username=cockatrice&repository=cockatrice - I'll add https for sure.

Comment (ZeldaZach) [inline README.md]: If you want it to open to that somsubhra site, then this is fine.

**Comment by tooomm** [comment]
It's the only site which I know of which offers to query and show the github api call reply.

We can also show only the picture and don't link at all! If you prefer.

**Comment by ZeldaZach** [comment]
I like, nice job

**Comment by tooomm** [comment]
Ok, added it in @ZeldaZach !

**Pull request merged**